crates/language_models/src/provider/ollama.rs

```diff
@@ -99,6 +101,7 @@ impl State {
                             None,
                             None,
                             Some(capabilities.supports_tools()),
+                            Some(capabilities.supports_tools()),
```

Comment:
This should be `capabilities.supports_vision()`

Reply:
Fixed this